plugins/BEdita/Core/src/Model/Table/ApplicationsTable.php

```diff
@@ -82,19 +82,19 @@ public function validationDefault(Validator $validator)
     {
         $validator
             ->integer('id')
-            ->allowEmpty('id', 'create')
+            ->allowEmptyString('id', null, 'create')
 
-            ->notEmpty('api_key')
+            ->notEmptyString('api_key')
             ->add('api_key', 'unique', ['rule' => 'validateUnique', 'provider' => 'table'])
 
             ->requirePresence('name', 'create')
-            ->notEmpty('name')
+            ->notEmptyString('name')
             ->add('name', 'unique', ['rule' => 'validateUnique', 'provider' => 'table'])
 
-            ->allowEmpty('description')
+            ->allowEmptyString('description')
 
             ->boolean('enabled')
-            ->notEmpty('enabled');
+            ->notEmptyString('enabled');
```

Comment:
`notEmptyString` for bool? also int?

Reply:
I was in doubt. Maybe it's better to add unit test for it.